pkg/runtime/wfengine/backends/actors/workflow_actor.go

```diff
@@ -352,7 +352,6 @@ func (wf *workflowActor) getWorkflowMetadata(ctx context.Context) (*api.Orchestr
 
 func (wf *workflowActor) getWorkflowState(ctx context.Context) (*workflowState, error) {
 	state, err := wf.loadInternalState(ctx)
-	wfLogger.Errorf("Workflow actor '%s': getWorkflowState, state: %s", wf.actorID, state)
```

Comment:
Can we change it to `wfLogger.Debugf` to match the rest of the code?

Reply:
Unless we extract non PII metadata from the struct I don't think it's required